transform/snowflake-dbt/models/events/hourly/cloud_pageview_events.sql

```diff
@@ -1,7 +1,7 @@
 {{config({
     "materialized": "incremental",
     "schema": "events",
-    "tags":"hourly"
+    "tags":["nightly"]
   })
```

Comment:
@sri-byte , sometimes we use "[nightly]", sometimes we use "nightly" - might be good to align on one format only.